### PR TITLE
Update glossary.yml

### PR DESCRIPTION
**Pull request by Taghreed-A**
## Author: 

- Taghreed

## Language: 

- Arabic

## Terms defined:
- confidence interval
- constant
- correlation

## Labels: 
- lang: ar

### CODE REVIEW

**Comment by Taghreed-A** [comment]
add the translations (in Arabic) of the terms confidence interval, constant, and correlation.

File: glossary.yml [hunk 1]
```diff
@@ -2593,6 +2593,11 @@
     term: በራስ የመተማመን ጊዜ
     def: >
       የስህተቱን ኅዳግ ከሚጠቁም ግምት ዙሪያ የሚገኝ ክልል፣ ከa ጋር ተደምሮ ትክክለኛው ዋጋ በዚህ ክልል ውስጥ የመውደቁ አጋጣሚ ሰፊ ነው።
+      
+  ar:
+    term: "فترة الثقة"
+    def: >
+    فترة الثقة تتكون من التقدير المعلمة وهامش الخطأ, وهي الفترة التي نثق بأن قيمة التقدير تكون داخل هذه الفترة
```
Comment (BatoolMM) [inline glossary.yml:2600]:
```suggestion
      فترة الثقة تتكون من التقدير المعلمة وهامش الخطأ, وهي الفترة التي نثق بأن قيمة التقدير تكون داخل هذه الفترة
```

Comment (BatoolMM) [reply glossary.yml:2600]: Can you fix this space?

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
يعطيك العافية!

**Review by BatoolMM — APPROVED** [review]

**Comment by elletjies** [comment]
Thank you for contributing to Glosario in 2022! We are looking forward to receiving more PRs from you during 2023!

**Comment by elletjies** [comment]
@all-contributors please add @Taalmasoud for translation

**Pull request merged**